### PR TITLE
feat: Add about page to fable test app

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Add about page from [Figma prototype](https://www.figma.com/proto/672zNYNzI3NqotZCccZjXT/Holidays-app?node-id=27-1233&t=yfEWqC9LQeFCt0Ic-0&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=13%3A824).

### CODE REVIEW

File: fable-test-app/src/pages/About.tsx [hunk 1]
```diff
@@ -1,13 +1,62 @@
 import React from 'react';
 
 // Components (internal)
-import { DateModified, Heading } from '../components';
+import { DateModified, Heading, Text } from '../components';
+import { GcdsCard, GcdsGrid } from '@cdssnc/gcds-components-react';
+import docsImage from '../assets/docs.png';
+import figmaImage from '../assets/figma.png';
+import githubImage from '../assets/github.png';
```
Comment (melaniebmn) [inline fable-test-app/src/pages/About.tsx:8]: If you add the images in the `public` folder instead of `src/assets`, you don't have to import them.

**Review by melaniebmn — CHANGES_REQUESTED** [review]
A small comment about the image import, it looks good otherwise.

File: fable-test-app/src/pages/About.tsx [hunk 2]
```diff
@@ -1,13 +1,59 @@
...
+        <GcdsCard
+          cardTitle="GitHub repo"
+          description="View our code in Github for all of our components."
+          cardTitleTag="h3"
+          badge="Code"
+          href="https://github.com/cds-snc/gcds-components"
+          img-src="github.png"
+          img-alt=""
+        />
+      </GcdsGrid>
```
Comment (melaniebmn) [inline fable-test-app/src/pages/About.tsx:48]: Sorry, another comment :D Should we create internal components for GcdsGrid and GcdsCard?

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Sorry, should have flagged this yesterday

**Review by daine — APPROVED** [review]
LGTM

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**